style.css

```diff
@@ -131,5 +145,190 @@ h1 {
   justify-content: space-between;
   align-items: center;
   width: 50%;
-  margin-bottom: 40%;
+  margin-bottom: 30%;
+}
+
+.project-main-wrapper {
+  background-color: #fff;
+  border-radius: 20px;
+  margin: auto 27px;
+}
+
+.project-container {
+  margin: auto 27px;
+}
+
+.postcollection {
+  margin-top: 50px;
+}
+
+.img-container {
+  width: 100%;
+}
+
+.snapshoot-img {
+  width: 100%;
+}
+
+.backend-dev {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  width: 90%;
+}
+
+.canopy {
+  width: 55px;
+  height: 16px;
+  font-size: 13px;
+  font-weight: 600;
+  font-style: normal;
+  line-height: 16px;
+  color: #344563;
+}
+
+.backend {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.access-project {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  height: 76px;
+  color: #344563;
+}
+
+.hypertext {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  text-align: center;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 10px 10px;
+  border: none;
+  padding: 5px 20px;
+}
+
+.cascading {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  text-align: center;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 10px 10px;
+  border: none;
+  padding: 5px 10px;
+}
+
+.javascript {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  text-align: center;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 10px 10px;
+  border: none;
+  padding: 5px 25px;
+  margin-bottom: 10px;
+}
+
+.project-review {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.my-qualities {
+  font-size: 16px;
+}
+
+.programming-skills {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.about-me {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-top: 20px;
+}
+
+.project-main-wrapperrr {
+  background-color: #fff;
+  margin-top: 60px;
+  border-top-right-radius: 80px;
+}
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. Kindly remove the top-right-border radius property from this container. :eyes: :eyes: 
Your design
![image](https://user-images.githubusercontent.com/98363075/198752357-c07e7afd-d2b3-40e5-ad74-a67756fd0f84.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/198752381-c9ab1e26-3850-4bbe-85b2-dce003d5ee18.png)